scripts/images/rancher-images.txt

```diff
@@ -4,27 +4,25 @@ docker.io/rancher/kubectl:v1.29.2
 docker.io/rancher/kubectl:v1.33.1
 docker.io/rancher/mirrored-cluster-api-controller:v1.10.2
 docker.io/rancher/mirrored-fluent-fluent-bit:3.1.8
-docker.io/rancher/mirrored-grafana-grafana:11.1.0
-docker.io/rancher/mirrored-ingress-nginx-kube-webhook-certgen:v1.4.3
-docker.io/rancher/mirrored-kiwigrid-k8s-sidecar:1.27.4
-docker.io/rancher/mirrored-kube-state-metrics-kube-state-metrics:v2.12.0
 docker.io/rancher/mirrored-kube-logging-fluentd:v1.16-4.10-full
 docker.io/rancher/mirrored-kube-logging-logging-operator:4.10.0
 docker.io/rancher/mirrored-kube-logging-config-reloader:v0.0.6
-docker.io/rancher/mirrored-library-busybox:1.31.1
-docker.io/rancher/mirrored-library-busybox:1.36.1
+docker.io/rancher/mirrored-grafana-grafana:11.5.5
+docker.io/rancher/mirrored-ingress-nginx-kube-webhook-certgen:v1.5.1
+docker.io/rancher/mirrored-kiwigrid-k8s-sidecar:1.30.0
+docker.io/rancher/mirrored-kube-state-metrics-kube-state-metrics:v2.15.0
 docker.io/rancher/mirrored-library-busybox:1.37.0
-docker.io/rancher/mirrored-library-nginx:1.24.0-alpine
+docker.io/rancher/mirrored-library-nginx:1.27.2-alpine
 docker.io/rancher/mirrored-prometheus-adapter-prometheus-adapter:v0.12.0
-docker.io/rancher/mirrored-prometheus-alertmanager:v0.27.0
-docker.io/rancher/mirrored-prometheus-node-exporter:v1.8.2
-docker.io/rancher/mirrored-prometheus-operator-prometheus-config-reloader:v0.75.1
-docker.io/rancher/mirrored-prometheus-operator-prometheus-operator:v0.75.1
-docker.io/rancher/mirrored-prometheus-operator-admission-webhook:v0.75.1
-docker.io/rancher/mirrored-prometheus-prometheus:v2.53.1
+docker.io/rancher/mirrored-prometheus-alertmanager:v0.28.1
+docker.io/rancher/mirrored-prometheus-node-exporter:v1.9.0
+docker.io/rancher/mirrored-prometheus-operator-prometheus-config-reloader:v0.80.1
+docker.io/rancher/mirrored-prometheus-operator-prometheus-operator:v0.80.1
+docker.io/rancher/mirrored-prometheus-operator-admission-webhook:v0.80.1
+docker.io/rancher/prom-prometheus:v3.2.1
+docker.io/rancher/kuberlr-kubectl:v5.0.0
```

Comment:
ntoe: `rancher-monitoring` chart uses `rancher/kuberlr-kubectl` to replace the original `rancher/shell`

but the `rancher-monitoring-crd` still uses the `rancher/shell`